src/account/default.ts

```diff
@@ -228,6 +233,34 @@ export class Account extends Provider implements AccountInterface {
     );
   }
 
+  public async deploy(
+    {
+      classHash,
+      salt,
+      unique = true,
+      callData = [],
+      isDevnet = false,
+    }: UniversalDeployerContractPayload,
+    transactionsDetail: InvocationsDetails = {}
+  ): Promise<any> {
```

Comment:
@tabaktoni mentioned about supporting multicalls!

Reply:
Great idea!

Reply:
Multicall can't be supported before 0.10.2 because we can't estimate fee of a transaction executed by an undeployed account.

Reply:
So I think we can have it but not all use cases will work, like the one you mentioned?

Reply:
But one could do multicall deploy true UDC with transactions like deploying Multiple contracts if he already has deployed account. Also i need to figure out how can one chain different transactions? If we can support that (unrelated to this PR)

Reply:
Yes, 

> So I think we can have it but not all use cases will work, like the one you mentioned?

Yes, if the transactions uses an account that's already deployed, it will work.